CHANGELOG.md

```diff
@@ -4,12 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
 
-## [0.0.71] - 2021-09-09
```

Comment:
The develop branch is currently awaiting a deploy to master. I've also branched off of the current state of the develop branch (with the release formatting ready to go). I've removed it here because it'll be added back in when doing the release.